src/Entity/Item.php

```diff
@@ -319,4 +319,16 @@ public function equals( EntityDocument $entity ) {
 		       && $this->statements->equals( $entity->statements );
 	}
 
+	/**
+	 * @see EntityDocument::__clone
+	 *
+	 * @since 5.0
+	 */
+	public function __clone() {
+		$this->fingerprint = clone $this->fingerprint;
+		// SiteLinkList is mutable, but SiteLink is not. No deeper cloning necesarry.
```

Comment:
There is nothing in `SiteLinkList` where this comment could be placed. I can't comment on non-existing code.